tests/isotope_test.py

```diff
@@ -201,7 +201,7 @@ def test_isotope_str(iso_str, sym, A, m):
 
 
 ISOTOPE_PROPERTIES = [
-    ('H-3', (3.888E8, False, None, '1/2+', 0., None, [['B-'], [100.]],)),
+    ('H-3', (3.888E8, False, None, '1/2+', 0., 14.94980957, [['B-'], [100.]],)),
```

Comment:
That's the mass excess, and apparently the database used to have an empty entry for it.